cmd/snowflake/full.go

```diff
@@ -119,12 +119,25 @@ func newFullCmd() *cobra.Command {
 				panic(err)
 			}
 
+			db, err := tidbsql.OpenTiDB(&tidbConfigFromCli)
+			if err != nil {
+				panic(err)
+			}
+			defer db.Close()
+
 			// get current tso
-			startTSO, err := tidbsql.GetCurrentTSO(&tidbConfigFromCli)
+			startTSO, err := tidbsql.GetCurrentTSO(db)
 			if err != nil {
 				panic(err)
 			}
 
+			// enlarge gc duration
+			if err = tidbsql.EnlargeGCDuration(db); err != nil {
+				panic(err)
+			}
+			// make sure reset gc duration
+			defer tidbsql.ResetGCDuration(db)
```

Comment:
Not sure but, this means if the application is exited unexpectedly, then long GC duration is kept (which is a serious problem). Do you have some ideas?

Reply:
no.....